packages/components/app/styles/components/pagination-bar.scss

```diff
@@ -2,4 +2,142 @@
 // PAGINATIONBAR
 //
 
-// .hds-pagination-bar {}
+@use "../mixins/focus-ring" as *;
+
+// * Pagination Parent (wrapper) component
+// ----------------------------------------
+.hds-pagination-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 36px;
+  color: var(--token-color-foreground-primary);
+  // font-size: var(--token-typography-body-300-font-size); // = 1rem, not same as 13px, TODO: investigate
+  font-size: 13px;
```

Comment:
I put back the hard-coded font-size for now because the token wasn't rendering as the correct size. I will clean up and refine the styling as I continue working on this in upcoming PRs @cveigt (This initial PR is meant to focus on overall structure and getting the basic visual look into place vs. the final cleaned up code.)

Reply:
Makes sense. Thanks for clarifying